src/store/domains/ui.ts

```diff
@@ -70,15 +70,25 @@ export default class UI implements UIStoreInterface, RoutingObservableInterface
           this.setOverviewViewType(UIOverviewViewType.TABLE);
         }
       }
-    )
+    );
+
+    reaction(
+      () => this.sidebarContent,
+      () => {
+        this.fetchForCurrentSideBarContent();
+      }
+    );
   }
 
   /* Computed values */
 
   get limitedToOverviews(): UIOverviewViewType[] {
     // We want to restrict the OverviewType to table,
     //  when the literature-references artifact is selected by the user
-    if (this.artifactKind === UIArtifactKind.LITERATURE_REFERENCES) {
+    if (
```

Comment:
Die Limitierung der Übersichtsansicht soll nicht in der Sammlungsansicht greifen.